tutorials/vpc-private-gateway-autoconfiguration/index.mdx

```diff
@@ -0,0 +1,179 @@
+
+2. The Public Gateway creation wizard displays. Enter the details of the new Public Gateway: 
+
+  - Choose the Availability Zone of the Public Gateway.
+  - Select the Public Gateway offer
+  - Select the Public Gateway IP address. To create a new IP, select **Allocate new IP** in the drop-down list. 
+  - Enter a name and optional tags for the Public Gateway
+
+3. Click **Create a Public Gateway**. The list of your Public Gateways displays.
+
+  <Lightbox src="scaleway_vpc_public_gateway_list.png" alt="" />
+
+4. Click **Manage** to display the Public Gateway Information. 
+
+5. Click **+ Attach a new Private Network** to attach the previously created Private Network to the Public Gateway.
+
+6. Select the Private Network from the drop-down list, then click **Attach Private Network**:
+
+  <Lightbox src="scaleway_vpc_public_gateway_attach_network.png" alt="" /> 
+
+7. Check if the IP address for the Private Network has been assigned to the instance using the DHCP server of the Private Gateway by running the `ip a` command: 
+
+  ```
+  3: ens5: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UP group default qlen 1000
+    link/ether 02:00:00:00:5a:d8 brd ff:ff:ff:ff:ff:ff
+    inet 192.168.0.2/24 brd 192.168.0.255 scope global dynamic ens5
+       valid_lft 3513sec preferred_lft 3513sec
+    inet6 fe80::ff:fe00:5bea/64 scope link 
+       valid_lft forever preferred_lft forever
+  ```
+
+  You can see the private IP address `192.168.0.2` has been configured automatically on the interface.
+
+Note that, once the instance is added to the Private Network and auto configured through DHCP, it cannot be accessed through its public IPv4 address anymore. Later in this tutorial, we explain how to configure port forwarding to access your instances through the Public Gateway's public IPv4 address.
+
+## Configuring port forwarding 
+
+Port forwarding allows to configure a mapping of ports on the IP address of the Public Gateway to IP addresses and ports inside the Private Network. This allows to access services on Virtual Instances in a Private Network without the need for a public IP address on each of them. 
+
+1. Click **VPC** in the **Network** section of the side menu in the Scaleway console. The VPC configuration displays.
+
+2. Click the **Public Gateways** tab. The list of your Public Gateways displays. 
+
+3. Click <Icon name="more" /> > **More Info** to display the Public Gateway information. 
+
+4. Click the **DHCP** tab. Then click **View DHCP Status** to display the list of devices autoconfigured by the DHCP server: 
+
+  <Lightbox src="scaleway_vpc_dhcp_status.png" />
+
+  <Message type="tip">
+
+  As you can notice in the screenshot above, both entries have the **dynamic** type. This means the DHCP server has dynamically assigned an IP address to each device. The assigned IP address may change and the port forwarding will stop working. To avoid this, configure a **static** lease. 
```

Comment:
```suggestion
  As you can see in the screenshot above, both entries have the **dynamic** type. This means the DHCP server has dynamically assigned an IP address to each device. The assigned IP address may change and the port forwarding will stop working. To avoid this, configure a **static** lease. 
```